<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>943</width>
    <height>714</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>pyAgdas</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>20</width>
      <height>281</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Project:</string>
    </property>
   </widget>
   <widget class="QLabel" name="name">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="check">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>943</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile_s">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="newProject"/>
    <addaction name="closeProject"/>
   </widget>
   <widget class="QMenu" name="menuData">
    <property name="title">
     <string>Data</string>
    </property>
    <addaction name="computing"/>
    <addaction name="viewdata"/>
    <addaction name="actionComparison"/>
   </widget>
   <addaction name="menuFile_s"/>
   <addaction name="menuData"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="newProject">
   <property name="text">
    <string>New project</string>
   </property>
  </action>
  <action name="computing">
   <property name="text">
    <string>Process data</string>
   </property>
  </action>
  <action name="Import_data">
   <property name="text">
    <string>Import all files</string>
   </property>
  </action>
  <action name="viewdata">
   <property name="text">
    <string>View data</string>
   </property>
  </action>
  <action name="viewGraphs">
   <property name="text">
    <string>View graphs</string>
   </property>
  </action>
  <action name="closeProject">
   <property name="text">
    <string>Close project</string>
   </property>
  </action>
  <action name="openProject">
   <property name="text">
    <string>Open project</string>
   </property>
  </action>
  <action name="actionComparison">
   <property name="text">
    <string>Comparison</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
